.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: ci
+on:
+  push:
+    branches: ["*"]
```

Comment:
```suggestion
on:
  pull_request:
  push:
    branches:
      - main
```
I think this describes better our use case . Do we need to build on every push ? In any case it will build for PR

Reply:
This best matched the previous flow, but I agree this makes more sense. Thanks!